src/compiler/scanner.ts

```diff
@@ -359,7 +359,7 @@ namespace ts {
 
     /* @internal */
     export function computeLineStarts(text: string): number[] {
-        const result: number[] = new Array();
+        const result: number[] = [];

```

Comment:
Maybe there was some reason for using `new Array` here? But performance doesn't seem to have changed.

Reply:
I don't think so. It's been there since the first commit of the TypeScript compiler.